inventory/redhat.cf

```diff
@@ -12,11 +12,13 @@ bundle common inventory_redhat
       comment => "derived from Red Hat",
       meta => { "inventory", "attribute_name=none" };
 
+      "inventory_redhat_have_python_symlink" expression => fileexists("/var/cfengine/bin/python");
+
+    inventory_redhat_have_python_symlink::
       "cfe_yum_package_module_supported" -> { "CFE-2602" }
         comment => "Here we see if the version of python found is acceptable for
                     the yum package module. We use this guard to prevent errors
                     related to missing python modules.",
-        expression => returnszero("python -V 2>&1 | grep ^Python | cut -d' ' -f 2 | ( IFS=. read v1 v2 v3 ; [ $v1 -ge 3 ] || [ $v1 -eq 2 -a $v2 -ge 4 ] )",
+        expression => returnszero("/var/cfengine/bin/python -V 2>&1 | grep ^Python | cut -d' ' -f 2 | ( IFS=. read v1 v2 v3 ; [ $v1 -ge 3 ] || [ $v1 -eq 2 -a $v2 -ge 4 ] )",
```

Comment:
What about using `$(sys.bindir)` instead of `/var/cfengine/bin` hard coded?

Reply:
We didn't want to render the package modules from templates, remember? If we do that, we don't need the symlink at all. What we could do is to use `@BINDIR@` and replace it when running `./configure`.

Reply:
Yeah, actually, that's what I was thinking about when I said it. Move this file to a .in and whatever automake magic to have it expand.

Reply:
Oh thats what I meant when I was commenting on the module itself. Here though, we are inside the policy, and so we could use sys.bindir

Reply:
Yes, I got that. :) But it needs to come from the same single source, which is what `@BINDIR@` from autotools is now.

Alright, I'll do this change.